src/nvim/tui/tui.c

```diff
@@ -208,10 +208,12 @@ static void try_resize(Event ev)
 static void sigwinch_cb(uv_signal_t *handle, int signum)
 {
   // Queue the event because resizing can result in recursive event_poll calls
+  /* FIXME(blueyed): this was changed to use a deferred event to fix #2322,
+   *                 without really knowing why it is necessary. */
```

Comment:
Please make this a C++ style comment, like the one above it, and declarative/present tense. Maybe...
```c
// FIXME(blueyed): The TUI does not resize properly when not deferred for unknown reasons. (#2322)
```
Not sure if this affects remote GUIs or not.